apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -378,6 +397,7 @@ export class ApplicationController {
     @AuthorizationHeader() authorization: string,
     @NationalId() nationalId: string,
   ): Promise<ApplicationResponseDto> {
+    await new Promise((resolve) => setTimeout(resolve, 1000))
```

Comment:
Still needed?

Reply:
Definitely not, should have removed. Thanks.